config/initializers/secure_headers.rb

```diff
@@ -6,7 +6,7 @@
     config.x_xss_protection = "1; mode=block"
     config.csp = {
       :report_only => false,
-      :default_src => ["'self'"],
+      :default_src => ["'self'", 'https://fonts.gstatic.com'],
```

Comment:
should this be a new `font_src` directive?
```suggestion
      :default_src => ["'self'"],
      :font_src => ['https://fonts.gstatic.com'],
```

Reply:
Yes we can use font_src, if we don't have font_src, it will fallback to default_src, let me know if you want me to add font_src.

Reply:
@kavyanekkalapu from the few places I read on the web, a separate `font_src` directive seems like the standard approach.  See https://rorsecurity.info/portfolio/content-security-policy-csp-strategy, https://bauland42.com/ruby-on-rails-content-security-policy-csp/, and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/font-src